root/types/editor.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * @flow strict
+ * Copyright (C) 2017 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+/*
+ * Types are in alphabetical order.
+ *
+ * The definitions in this file are intended to model the output of the
+ * TO_JSON methods under lib/MusicBrainz/Server/Entity/, those are precisely
+ * how data is serialized for us.
+ */
+
+declare type ActiveEditorPreferencesT = {
+  +datetime_format: string,
+  +timezone: string,
+};
+
+/*
+ * Only used in cases where the editor being sanitized is logged in;
+ * exposes some preferences and misc. metadata.
+ */
+declare type ActiveEditorT = {
+  ...EntityRoleT<'editor'>,
+  +gravatar: string,
+  +has_confirmed_email_address: boolean,
+  +name: string,
+  +preferences: ActiveEditorPreferencesT,
+};
+
+declare type EditorLanguageT = {
+  +fluency: FluencyT,
+  +language: LanguageT,
+};
+
+declare type EditorT = {
+  ...EntityRoleT<'editor'>,
+  +deleted: boolean,
+  +gravatar: string,
+  +name: string,
+};
+
+declare type FluencyT =
+  | 'basic'
+  | 'intermediate'
+  | 'advanced'
+  | 'native'
+  ;
```

Comment:
I think you recently fixed these so let's not re-add them :)
```suggestion
  | 'native';
```